diff --git a/js/ui/panel.js b/js/ui/panel.js
index 94dffda73..8decf5629 100644
--- a/js/ui/panel.js
+++ b/js/ui/panel.js
@@ -361,7 +361,6 @@ class QuickSettings extends PanelMenu.Button {
         this._location = new imports.ui.status.location.Indicator();
         this._thunderbolt = new imports.ui.status.thunderbolt.Indicator();
         this._nightLight = new imports.ui.status.nightLight.Indicator();
-        this._darkMode = new imports.ui.status.darkMode.Indicator();
         this._powerProfiles = new imports.ui.status.powerProfiles.Indicator();
         this._rfkill = new imports.ui.status.rfkill.Indicator();
         this._autoRotate = new imports.ui.status.autoRotate.Indicator();
@@ -374,7 +373,6 @@ class QuickSettings extends PanelMenu.Button {
         this._indicators.add_child(this._nightLight);
         if (this._network)
             this._indicators.add_child(this._network);
-        this._indicators.add_child(this._darkMode);
         this._indicators.add_child(this._powerProfiles);
         if (this._bluetooth)
             this._indicators.add_child(this._bluetooth);
@@ -397,7 +395,6 @@ class QuickSettings extends PanelMenu.Button {
             this._addItems(this._bluetooth.quickSettingsItems);
         this._addItems(this._powerProfiles.quickSettingsItems);
         this._addItems(this._nightLight.quickSettingsItems);
-        this._addItems(this._darkMode.quickSettingsItems);
         this._addItems(this._rfkill.quickSettingsItems);
         this._addItems(this._autoRotate.quickSettingsItems);
         this._addItems(this._unsafeMode.quickSettingsItems);
